### PR TITLE
DSP-223: deconstruct /not-arrived translations

### PR DESCRIPTION
**Pull request by JoeChapman**
- separate default.json into src/
- install 0.0.2 hof-transpiler
- use hof-transpiler to build default.json

### CODE REVIEW

File: apps/not-arrived/translations/en/default.json [hunk 1]
```diff
@@ -1,1038 +1,358 @@
-{
```
Comment (easternbloc) [inline apps/not-arrived/translations/en/default.json]: perhaps we should just delete the default.json file? I added hof-transpile as part of the postinstall in another PR so we can assume this is always built?

Comment (JoeChapman) [inline apps/not-arrived/translations/en/default.json]: I think that should be in another PR that we raise after we've fixed all the translations. Otherwise, we'll be ignoring default.json files that have not be built by hof and that will break the build

Comment (easternbloc) [inline apps/not-arrived/translations/en/default.json]: I'll get my coat.
